portfolio/src/main/webapp/scriptFetch.js

```diff
@@ -0,0 +1,18 @@
+/* Gets a string to display on page  */
+async function getRandomString() {
+
+  //const textFromResponse = "This is a random string";
+
+  // Send a request to /random-string.
+  const responseFromServer = await fetch('/random-string');
+    
+  // Parse the response as JSON.
+  const messages = await responseFromServer.json();
+
+    const randomString = words[Math.floor(Math.random() * messages.length)];
```

Comment:
Why indented?

Reply:
Ideally you can follow Google coding style: https://google.github.io/styleguide/jsguide.html